app/src/main/java/io/github/pietrocaselani/moviestraker/tmdb/TMDB.kt

```diff
@@ -0,0 +1,63 @@
+package io.github.pietrocaselani.moviestraker.tmdb
+
+import com.google.gson.GsonBuilder
+import com.jakewharton.retrofit2.adapter.rxjava2.RxJava2CallAdapterFactory
+import io.github.pietrocaselani.moviestraker.tmdb.services.Configuration
+import io.github.pietrocaselani.moviestraker.tmdb.services.Genres
+import io.github.pietrocaselani.moviestraker.tmdb.services.Movies
+import io.github.pietrocaselani.moviestraker.tmdb.services.Search
+import okhttp3.OkHttpClient
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+
+/**
+ * Created by pc on 24/06/17.
+ */
+open class TMDB(private val apiKey: String) {
```

Comment:
I designed the `tmdb` package in this way so it can be moved to another package and used as a library. This class is `open` so the application can extend it and then provide modifications to `retrofit`, `okhttp` or `gson`.